guide/english/ruby/common-array-methods/index.md

```diff
@@ -185,22 +185,15 @@ end
 ```
 
 #### .map
-The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements.
+The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements:
 
 ``` ruby
 array.map { |element| element * 2 }
-  puts element
-end
-=> 
-0
-2
-4
-6
-8
+=> [0, 2, 4, 6, 8]
 ```
 
 #### .uniq
-The .uniq method takes in an array containing duplicate elements, and returns a copy of the array containing only unique elements--any duplicate elements are removed from the array.
+The .uniq method takes in an array containing duplicate elements, and returns a copy of the array containing only unique elements--any duplicate elements are removed from the array:
```

Comment:
same as the suggestion for the previous definition, use back-ticks for code methods and variables to distinguish themselves from normal text